src/pages/billing/Project.js

```diff
@@ -368,6 +381,19 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
             }
           }
         }, [icon('edit', { size: 12 })]),
+        h(Link, {
+          tooltip: 'Remove Billing Account',
+          style: { marginLeft: '0.5rem' },
+          baseColor: colors.danger,
+          onClick: async () => {
+            if (Auth.hasBillingScope()) {
+              setShowBillingRemovalModal(true)
+            } else {
+              await authorizeAndLoadAccounts()
+              setShowBillingRemovalModal(Auth.hasBillingScope())
```

Comment:
do billing project users have billing scopes? / should billing project users be able to remove the billing account from a billing project?

Reply:
I don't know if users have billing scopes, but currently they don't have access to the project details page. (All 'shared with me' projects are disabled.)